pkg/testidentification/ocp_variants_test.go

```diff
@@ -50,7 +52,7 @@ func Test_openshiftVariants_IdentifyVariants(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			v := openshiftVariants{}
-			if got := v.IdentifyVariants(tt.name, tt.release); !reflect.DeepEqual(got, tt.want) {
+			if got := v.IdentifyVariants(tt.name, tt.release, models.ClusterData{}); !reflect.DeepEqual(got, tt.want) {
```

Comment:
Could we add a couple test cases for the cluster data? I think it'd be pretty straight forward to add a cluster data field to the tests struct above

Reply:
Yeah, good point and nice catch.  

Wondering if the scan in gcs is making the load data time out in the e2e now... didn't see that before I renamed the file but can't imagine that would do it.